mods/ts/chrome.yaml

```diff
@@ -1,394 +1,300 @@
+scrollup-buttons-nod-hover: chrome.png
+	background: 547,364,34,27
+scrollup-buttons-nod-pressed: chrome.png
+	background: 582,364,34,27
+scrollup-buttons-nod-highlighted: chrome.png
+	background: 547,364,34,27
+scrollup-buttons-nod-highlighted-hover: chrome.png
+	background: 547,364,34,27
+scrollup-buttons-nod-highlighted-pressed: chrome.png
+	background: 582,364,34,27
+scrollup-button-nod-highlighted-disabled: chrome.png
+	background: 512,364,34,27
+scrollup-buttons-nod-disabled: chrome.png
+	background: 512,364,34,27
+	
+scrolldown-buttons-nod: chrome.png
+	background: 617,364,34,27
+scrolldown-buttons-nod-hover: chrome.png
+	background: 652,364,34,27
+scrolldown-buttons-nod-pressed: chrome.png
+	background: 687,364,34,27
+scrolldown-buttons-nod-highlighted: chrome.png
+	background: 652,364,34,27
+scrolldown-buttons-nod-highlighted-hover: chrome.png
+	background: 652,364,34,27
+scrolldown-buttons-nod-highlighted-pressed: chrome.png
+	background: 687,364,34,27
+scrolldown-button-nod-highlighted-disabled: chrome.png
+	background: 617,364,34,27
+scrolldown-buttons-nod-disabled: chrome.png
+	background: 617,364,34,27
+
+radar-nod: chrome.png
+	insignia: 513,392,164,119
+
+production-icons-nod: chrome.png
+	building: 928,0,16,16
+	building-disabled: 944,0,16,16
+	building-alert: 960,0,16,16
+	defense: 928,16,16,16
+	defense-disabled: 944,16,16,16
+	defense-alert: 960,16,16,16		
+	infantry: 928,32,16,16
+	infantry-disabled: 944,32,16,16
+	infantry-alert: 960,32,16,16
+	vehicle: 928,96,16,16
+	vehicle-disabled: 944,96,16,16
+	vehicle-alert: 960,96,16,16
+	aircraft: 928,64,16,16
+	aircraft-disabled: 944,64,16,16
+	aircraft-alert: 960,64,16,16
+
+order-icons-nod: chrome.png
+	debug: 808,289,30,31
+	debug-disabled: 839,289,30,31
+	debug-active: 808,808,30,31
+	repair: 808,321,30,31
+	repair-disabled: 839,321,30,31
+	repair-active: 808,321,30,31
+	sell: 808,353,30,31
+	sell-disabled: 839,353,30,31
+	sell-active: 808,353,30,31
+	options: 808,385,30,31
+	options-disabled: 839,385,30,31
+	options-active: 808,385,30,31  
+	beacon: 808,417,30,31
+	beacon-disabled: 839,417,30,31
+	beacon-active: 808,417,30,31
+	power: 808,449,30,31
+	power-disabled: 839,449,30,31
+	power-active: 808,449,30,31
+	diplomacy: 808,481,30,31
+	diplomacy-disabled: 839,481,30,31
+	diplomacy-active: 808,481,30,31
+
+# make this paragraph AddRaceSuffixLogic compatible
+sidebar-bits-nod: chrome.png
+	production-tooltip-time: 928,208,16,16
+	production-tooltip-power: 992,160,16,16
+	production-tooltip-cost: 992,144,16,16
+	production-iconoverlay: 168,392,64,48
+
+power-icons-nod: chrome.png
+	power-normal: 992,160,16,16
+	power-critical: 1008,160,16,16
+
+cash-icons-nod: chrome.png
+	cash-normal: 992,144,16,16
+	cash-critical: 1008,144,16,16
+
+# ----------------------------------------------------------------------
+# GENERIC
+# ----------------------------------------------------------------------
+	
+# make this paragraph AddRaceSuffixLogic compatible
 sidebar-bits: chrome.png
```

Comment:
I am a bit confused if this is still required. Maybe the comment on top is just misleading.